css/styles.css

```diff
@@ -0,0 +1,574 @@
+    font-weight: normal;
+    font-style: normal;
+}
+
+
+@font-face {
+    font-family: 'News Gothic MT Std';
+    src: url('fonts/NewsGothicMTStd-Bold.eot');
+    src: url('fonts/NewsGothicMTStd-Bold.eot?#iefix') format('embedded-opentype'),
+        url('fonts/NewsGothicMTStd-Bold.woff2') format('woff2'),
+        url('fonts/NewsGothicMTStd-Bold.woff') format('woff'),
+        url('fonts/NewsGothicMTStd-Bold.ttf') format('truetype'),
+        url('fonts/NewsGothicMTStd-Bold.svg#NewsGothicMTStd-Bold') format('svg');
+    font-weight: bold;
+    font-style: normal;
+}
+
+
+@font-face {
+    font-family: 'Baskerville MT Std';
+    src: url('fonts/BaskervilleMTStd-Bold.eot');
+    src: url('fonts/BaskervilleMTStd-Bold.eot?#iefix') format('embedded-opentype'),
+        url('fonts/BaskervilleMTStd-Bold.woff2') format('woff2'),
+        url('fonts/BaskervilleMTStd-Bold.woff') format('woff'),
+        url('fonts/BaskervilleMTStd-Bold.ttf') format('truetype'),
+        url('fonts/BaskervilleMTStd-Bold.svg#BaskervilleMTStd-Bold') format('svg');
+    font-weight: bold;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Baskerville MT Std';
+    src: url('fonts/BaskervilleMTStd-Italic.eot');
+    src: url('fonts/BaskervilleMTStd-Italic.eot?#iefix') format('embedded-opentype'),
+        url('fonts/BaskervilleMTStd-Italic.woff2') format('woff2'),
+        url('fonts/BaskervilleMTStd-Italic.woff') format('woff'),
+        url('fonts/BaskervilleMTStd-Italic.ttf') format('truetype'),
+        url('fonts/BaskervilleMTStd-Italic.svg#BaskervilleMTStd-Italic') format('svg');
+    font-weight: normal;
+    font-style: italic;
+}
+
+@font-face {
+    font-family: 'Baskerville MT Semi';
+    src: url('fonts/BaskervilleMTStd-SemiBdIt.eot');
+    src: url('fonts/BaskervilleMTStd-SemiBdIt.eot?#iefix') format('embedded-opentype'),
+        url('fonts/BaskervilleMTStd-SemiBdIt.woff2') format('woff2'),
+        url('fonts/BaskervilleMTStd-SemiBdIt.woff') format('woff'),
+        url('fonts/BaskervilleMTStd-SemiBdIt.ttf') format('truetype'),
+        url('fonts/BaskervilleMTStd-SemiBdIt.svg#BaskervilleMTStd-SemiBdIt') format('svg');
+    font-weight: 600;
+    font-style: italic;
+}
+
+@font-face {
+    font-family: 'Baskerville MT Semi';
+    src: url('fonts/BaskervilleMTStd-SemiBd.eot');
+    src: url('fonts/BaskervilleMTStd-SemiBd.eot?#iefix') format('embedded-opentype'),
+        url('fonts/BaskervilleMTStd-SemiBd.woff2') format('woff2'),
+        url('fonts/BaskervilleMTStd-SemiBd.woff') format('woff'),
+        url('fonts/BaskervilleMTStd-SemiBd.ttf') format('truetype'),
+        url('fonts/BaskervilleMTStd-SemiBd.svg#BaskervilleMTStd-SemiBd') format('svg');
+    font-weight: 600;
+    font-style: normal;
+}
+
+
+body {
+	margin: 0;
+	font-family: 'News Gothic MT Std';
+	}
+
+nav {
+	width: 1200px;
+	margin: 0;
+	overflow: auto;
+}
+
+a {
+	text-decoration: none;
+	color: #000000;
+}
+
+.logoLink {
+	color: black;
+}
+
+.logoLink:hover {
```

Comment:
Does this need to be here. Does't seem to be doing anything. Are you going to use it later? :)